scripts/fetch_updated_release_notes_file.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Adyen N.V.
+#
+# This file is open source and available under the MIT license. See the LICENSE file for more info.
+#
+# Created by ozgur on 10/12/2024.
+#
+function fetch_updated_release_notes_file() {
```

Comment:
We already fetch the version name in `fetch_release_notes_version_name.sh` file. I would either pass the version name to this script or construct the file name in the action itself.